### PR TITLE
New version: Mu.Mu version 1.1.0-beta.7

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38611)

### CODE REVIEW

File: manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mu.Mu
+PackageVersion: 1.1.0-beta.7
```
Comment (ItzLevvie) [inline manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.1.0
```

File: manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Mu.Mu
+PackageVersion: 1.1.0-beta.7
```
Comment (ItzLevvie) [inline manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.0
```

File: manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Mu.Mu
+PackageVersion: 1.1.0-beta.7
```
Comment (ItzLevvie) [inline manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.yaml:5]:
```suggestion
PackageVersion: 1.1.0
```

File: manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/mu-editor/mu/releases/download/1.1.0-beta.7/Mu-Editor-Win64-1.1.0b7.msi
+  InstallerSha256: 775E9D963F0AE02DC98D0394FCB57BE85E202C0CD90EAD5B2DAE31294D8BA5E7
```
Comment (ItzLevvie) [inline manifests/m/Mu/Mu/1.1.0-beta.7/Mu.Mu.installer.yaml:18]:
```suggestion
  InstallerSha256: 775E9D963F0AE02DC98D0394FCB57BE85E202C0CD90EAD5B2DAE31294D8BA5E7
  ProductCode: "{2858D67E-5485-419C-A099-19EABD12F748}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]
VSCode online made it way harder to rename the folder than it needed to.. (Folders disappeared)

**Pull request merged**